lektor/admin/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "lektor",
   "version": "0.0.0",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
+    "create-react-class": "^15.6.3"
```

Comment:
It's a bit weird to have this as a normal dependency when everything else is a dev-dependency :o)

Reply:
Fixed in 2dbb05e - I think I must have run `--save` rather than `--save-dev`!